src/app/contact/contact.css

```diff
@@ -0,0 +1,47 @@
+.contact {
+  max-width: 750px;
+  margin-right: auto;
+  margin-bottom: 0;
+  margin-left: auto;
+}
+
+.contact h1 {
+  margin-top: 0;
+  margin-bottom: 20px;
+  text-align: center;
+  font-weight: 300;
+}
+
+.contact .form-group {
+  position: relative;
+  margin-bottom: 30px;
+}
+
+.contact .h4:first-child {
+  margin-top: 0;
+}
+
+.contact button {
+  margin-bottom: 30px;
+}
+
+.contact label,
+.contact label small {
+  font-weight: 300;
+}
+
+.contact a {
+  color: #006F3E;
+  color: rgba(0,111,62,.8);
```

Comment:
`color: #006F3E;` should be deleted since it is being overridden in the next line and never used